src/Sql/Sql/Properties/Resources.resx

```diff
@@ -631,4 +631,7 @@
   <data name="FailoverAzureSqlInstanceWarning" xml:space="preserve">
     <value>Are you sure you want to failover the Azure Sql Managed Instance '{0}'?</value>
   </data>
+  <data name="ServerNameInvalid" xml:space="preserve">
+    <value>Invalid server name: '{0}'. Server name can contain only lowercase letters, number.</value>
```

Comment:
Can update to this?

The server name '{0}' cannot be empty or null. The server name can only be made up of lowercase letters a-z, the numbers 0-9 and the hyphen. The hyphen may not lead or trail in the server name. Please fix the server name and retry. Please contact Microsoft support if the issue persists.

Reply:
Done